src/cli/strategies/git-cz.js

```diff
@@ -47,20 +47,31 @@ function gitCz(rawGitArgs, environment, adapterConfig) {
   let resolvedAdapterRootPath = findRoot(resolvedAdapterConfigPath);
   let prompter = getPrompter(adapterConfig.path);
 
-  isClean(process.cwd(), function(stagingIsClean){
+  isClean(process.cwd(), function(error, stagingIsClean){
+    if (error) {
+      throw error;
+    }
+
     if(stagingIsClean) {
-      console.error('Error: No files added to staging! Did you forget to run git add?')
-    } else {
-      // OH GOD IM SORRY FOR THIS SECTION
-      let adapterPackageJson = getParsedPackageJsonFromPath(resolvedAdapterRootPath);
-      let cliPackageJson = getParsedPackageJsonFromPath(environment.cliPath);
-      console.log(`cz-cli@${cliPackageJson.version}, ${adapterPackageJson.name}@${adapterPackageJson.version}\n`);
-      commit(sh, inquirer, process.cwd(), prompter, {args: parsedGitCzArgs, disableAppendPaths:true, emitData:true, quiet:false, retryLastCommit}, function() {
-        // console.log('commit happened');
-      });
-      
+      throw new Error('Error: No files added to staging! Did you forget to run git add?');
```

Comment:
You can remove the `Error: ` part from the start of the message, the error class will add that when casting to a string.